Features/Context/Users.php

```diff
@@ -36,6 +36,17 @@ public function iCreateUser(TableNode $users = null)
     }
 
     /**
+     * @When I go to (the) User :username page
+     */
+    public function goToUserPage($username)
+    {
+        $this->iAmOnPage('Users');
+        $this->waitWhileLoading();
+        $this->clickOnTreePath("$username $username");
```

Comment:
actually, won't this work solely for users created in behat (same first/last name)?
what about admin, etc?

Correct/long approach would be to load the User's first/last name then use those values.